### PR TITLE
fix662 Update man page to clarify rename overwrite

### PR DESCRIPTION
**Pull request by clanmills**

### CODE REVIEW

**Review by D4N — COMMENTED** [review]
Could you please change the commit message to reference #662 instead of #622? I can do that too.

File: src/exiv2.1 [hunk 1]
```diff
@@ -278,12 +278,47 @@ not rename the file (overrides \fB\-k\fP). This option is only used
...
+File 1/1: Stonehenge.jpg
+Renaming file to ./20150716_153854.jpg
+$ curl --silent -O http://clanmills.com/Stonehenge.jpg
+$ exiv2 --verbose --Force rename Stonehenge.jpg
+File 1/1: Stonehenge.jpg
+Renaming file to ./20150716_153854_1.jpg
+.sp 1
+.fi
+The 'rename' command will only overwrite files when option --force is used.  The option --Force is provided to avoid unintentional loss of valuable image files.
+.sp 1
```
Comment (D4N) [inline src/exiv2.1]: So it doesn't make a difference for 'extract' whether -f or -F is used?

File: src/exiv2.1 [hunk 2]
```diff
@@ -278,12 +278,47 @@ not rename the file (overrides \fB\-k\fP). This option is only used
...
+$ curl --silent -O http://clanmills.com/Stonehenge.jpg
+$ exiv2 --verbose --Force rename Stonehenge.jpg
+File 1/1: Stonehenge.jpg
+Renaming file to ./20150716_153854.jpg
+$ curl --silent -O http://clanmills.com/Stonehenge.jpg
+$ exiv2 --verbose --Force rename Stonehenge.jpg
+File 1/1: Stonehenge.jpg
+Renaming file to ./20150716_153854_1.jpg
+.sp 1
+.fi
```
Comment (D4N) [inline src/exiv2.1]: files when option --force -> files when the option --force

File: src/exiv2.1 [hunk 3]
```diff
@@ -278,12 +278,47 @@ not rename the file (overrides \fB\-k\fP). This option is only used
...
-.B \-f
-Do not prompt before overwriting existing files (force overwrite).
+.B \-f,\-F
+These options are used by commands 'rename' and 'extract' to determine file overwrite policy.  These options
+are usually combined with -v/--verbose to provide additional status output.
+.br
+.sp 1
+The options --force and --Force apply to the 'rename' command.  The 'extract' command treats --force and --Force as permission to overwrite.
+.br
+.sp 1
```
Comment (D4N) [inline src/exiv2.1]: Default behaviour is to prompt user. -> The default behaviour is to prompt the user.

File: src/exiv2.1 [hunk 4]
```diff
@@ -278,12 +278,47 @@ not rename the file (overrides \fB\-k\fP). This option is only used
 with the 'rename' action. Note: On Windows you may have to set the TZ
 environment variable for this option to work correctly.
 .TP
-.B \-f
-Do not prompt before overwriting existing files (force overwrite).
+.B \-f,\-F
```
Comment (D4N) [inline src/exiv2.1]: are used by commands 'rename' and 'extract' to determine file overwrite -> are used by the commands 'rename' and 'extract' to determine the file overwrite
('the' missing 2 times)

File: src/exiv2.1 (hunk 1 above)
Comment (clanmills) [reply src/exiv2.1]: For command `extract`, the behaviour of options `--force` and `--Force` are identical.

For command `rename`, the option `--force` grants permission to overwrite.

For command `rename`, the option `--Force` appends _N to the generated filename to ensure that image files are not overwritten.

**Comment by clanmills** [comment]
Thanks for fixing the 622/662 error.

The changes you request are tiny.  I can make changes, however I'm sure I will not get them exactly right.  To avoid a stream of updates, can you update and submit.

**Comment by D4N** [comment]
> To avoid a stream of updates, can you update and
> submit.

Done, I'll merge this once the CI is green.

**Review by D4N — APPROVED** [review]

**Pull request merged**